fizzBuzzRedux.js

```diff
@@ -0,0 +1,51 @@
+//Benton Green
+  function FizzBuzz(input1, input2){
+    var startNum;
+    var endNum;
+    this.input1 = input1;
+    this.input2 = input2;
+    
+    this.readFizzBuzz = function(startNum, endNum){
+      startingNum = startNum;
+      endingNum = endNum;
+    };
+
+    this.writeFizzBuzz = function() {
+      this.startingNum = startingNum; 
+      this.endingNum = endingNum + 1; //added one to stopNum so that it would include ending number in list
+      
+      fbOutputList = document.getElementsByClassName('fizzBuzzOutput')[0];
```

Comment:
Don't hard-code the classname of the node you're ultimately writing your content to. The way this is called in your main.js file, you actually have a redundant lookup (since you're passing in `document.getElementsByClassName` to `writeToFizzBuzz`).